src/js/providers/tracks-mixin.js

```diff
@@ -123,16 +123,19 @@ define(['utils/underscore',
             }
         }
 
-        if (this.renderNatively) {
-            // Only bind and set this.textTrackChangeHandler once so that removeEventListener works
-            this.textTrackChangeHandler = this.textTrackChangeHandler || textTrackChangeHandler.bind(this);
-            this.addTracksListener(this.video.textTracks, 'change', this.textTrackChangeHandler);
-
-            if (utils.isEdge() || utils.isFF() || utils.isSafari()) {
-                // Listen for TextTracks added to the videotag after the onloadeddata event in Edge and Firefox
-                this.addTrackHandler = this.addTrackHandler || addTrackHandler.bind(this);
-                this.addTracksListener(this.video.textTracks, 'addtrack', this.addTrackHandler);
-            }
+        // Only bind and set this.textTrackChangeHandler once so that removeEventListener works
+        this.textTrackChangeHandler = this.textTrackChangeHandler || textTrackChangeHandler.bind(this);
```

Comment:
We don't need to listen for track changes coming from the browser when using the player's renderer. That should only come from the API: https://github.com/jwplayer/jwplayer/blob/master/src/js/controller/controller.js#L683